src/pages/Patch.tsx

```diff
@@ -1,22 +1,77 @@
 import React from "react";
+import { useParams } from "react-router-dom";
 import { BreadCrumb } from "components/Breadcrumb";
-import { BigTitle } from "components/styles/Typography";
-import { PageWrapper } from "components/styles/PageWrapper";
-import { SiderCard } from "components/styles/SiderCard";
-import { PageHeader } from "components/styles/PageHeader";
-import { PageContent, PageLayout, PageSider } from "components/styles/Layout";
+import { H1, H3, P2 } from "components/Typography";
+import {
+  PageWrapper,
+  SiderCard,
+  PageHeader,
+  StyledLink,
+  PageContent,
+  PageLayout,
+  PageSider
+} from "components/styles";
+import { Divider } from "components/styles/Divider";
+import { useQuery } from "@apollo/react-hooks";
+import { GET_PATCH, PatchQuery } from "gql/queries/patch";
 
 export const Patch = () => {
```

Comment:
I know this code change isn't part of your PR but it's missing the React.FC type

Reply:
if the compiler does not complain then it is not needed. `React.FC` component is required when component accepts props.